tests/daemon_test.py

```diff
@@ -78,26 +79,26 @@ def test_init_resource(tmpdir, sanlock_daemon):
 
 def test_format(tmpdir, sanlock_daemon):
     path = tmpdir.join("rindex")
-    size = 1024**2 * 3
+    size = MiB * 3
```

Comment:
While we fix this we should also change the order to:

    3 * MiB

Which is more natural.